src/UI/Components/InlineEditable/EditableTextField.tsx

```diff
@@ -53,5 +53,7 @@ const EditView: EditViewComponent = ({ value, onChange, onSubmit, label }) => (
 );
 
 const StaticView: StaticViewComponent = ({ value, ...props }) => (
-  <InlineValue {...props}>{value}</InlineValue>
+  <InlineValue role="textbox" {...props}>
```

Comment:
why are we setting the role here?

Reply:
axe assertion in Env Settings scenario was complaining that this div has aria-label but no role assigned

Reply:
yeah, but role textbox is for an input element, this is a div. It shouldn't have that role.

Reply:
I could not find better fitting role for this component, and it's a preview of value for input component but I understand your remark. I change the tag of the component

Reply:
I looked for patternfly Content component but this is also div without role, paragraph won't work  and I think this is closest role that could be there. I can create ticket to rework those components to be proper inputs with read-only view instead of what it currently is. what do you think about it?

Reply:
Why are we passing an aria-label to a div to begin with?

Reply:
sure make a ticket

Reply:
https://github.com/inmanta/web-console/issues/6328